src/tictactoe/lib/tictactoe_web/controllers/version_controller.ex

```diff
@@ -0,0 +1,12 @@
+defmodule TictactoeWeb.VersionController do
+  use TictactoeWeb, :controller
+
+  alias TictactoeWeb.Version
+
+  action_fallback TictactoeWeb.FallbackController
+
+  def index(conn, _params) do
+    result = %{version: "0.11"}
```

Comment:
It would be pretty cool if this contained
  * the git SHA
  * `-dirty` when there are local changes.

I really like this one!

Reply:
It is handy for various usecases. I'm not yet familiar with Elixir internals. Sometimes CI pipeline assigns version automatically by updating some config.
So I will think how to implement automatic versioning within hosting implementation.